help/destinations/catalog/crm/salesforce.md

```diff
@@ -8,11 +8,15 @@ exl-id: bd9cb656-d742-4a18-97a2-546d4056d093
 
 ## Overview {#overview}
 
-[Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
+[[!DNL Salesforce CRM]](https://www.salesforce.com/crm/) is a popular Customer Relationship Management (CRM) platform and supports the below:
+* [Leads](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_lead.htm) - A lead is the name of a person or company who may (or may not) be interested in the products or services you sell.
+* [Contacts](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_contact.htm) - A contact is an individual with whom one of your reps has established a relationship and has been qualified as a potential customer.	 
 
-This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts), which allows you to update identities within a segment into Salesforce CRM.
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [!DNL[Salesforce composite API]](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_sobjects_collections_update.htm) which supports yboth types.
```

Comment:
Couple of comments here: 

1. Please add a comma before `which`
2. both instead of yboth. Also, let's expand: `[...] which supports both types of profiles described above.`
3. Please double-check the link for !DNLSalesforce composite API. it doesn't look like it's rendering properly for me.